src/containers/Detail/components/TableDetail/index.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { Avatar, Table, Group, Text, ScrollArea } from '@mantine/core';
+import styles from './styles.module.scss';
+
+interface tableDetailProps {
```

Comment:
This interface should follow UpperCamelCase.

Reply:
done